pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -0,0 +1,99 @@
+ { stdenv, python }:
+let
+  inherit (localPythonPackageSet) pkgs;
+  inherit (pkgs) fetchPypi buildPythonApplication;
+
+  localPythonPackageSet = python.override {
+    packageOverrides = self: super: rec {
+      colorama = super.colorama.overridePythonAttrs (oldAttrs: rec {
+        version = "0.3.7";
+
+        src = fetchPypi {
+          inherit (oldAttrs) pname;
+          inherit version;
+          sha256 = "0avqkn6362v7k2kg3afb35g4sfdvixjgy890clip4q174p9whhz0";
+        };
+      });
+
+      docker = super.docker.overridePythonAttrs (oldAttrs: rec {
```

Comment:
am I correct that you are now using a different package for `docker`? That is, `docker-py` instead of `docker`?

Reply:
It's the same package from the same repo (https://github.com/docker/docker-py) but since version 2.0.0 (the version in pkgs/development/python-modules/docker.nix is 2.5.1) they changed the name that you would use to install it with pip.

It's in the release notes here: https://github.com/docker/docker-py/releases/tag/2.0.0

> The name of the pip package is now docker (was: docker-py). New versions of this library will only be published as docker from now on.

Should I create a new derivation for it?